Ginger/GingerCoreNET/ActionsLib/ActSetVariableValue.cs

```diff
@@ -134,6 +136,49 @@ public override void Execute()
                 {
                     ((VariableDynamic)Var).ValueExpression = this.Value;
                 }
+                else if (Var.GetType() == typeof(VariableNumber))
+                {
+                    try
+                    {
+                        var varNumber = ((VariableNumber)Var);
+                        if (float.Parse(calculatedValue) >= float.Parse(varNumber.MinValue) && float.Parse(calculatedValue) <= float.Parse(varNumber.MaxValue))
+                        {
+                            varNumber.Value = calculatedValue;
+                        }
+                        else
+                        {
+                            Status = Amdocs.Ginger.CoreNET.Execution.eRunStatus.Failed;
```

Comment:
We should not set the status. Just update the Error field value. and Ginger runner knows to calculate the status based on error is empty or not

Reply:
changes pushed